TrailBlaze/View Controllers/RerunViewController.m

```diff
@@ -0,0 +1,92 @@
+//
+//  RerunViewController.m
+//  TrailBlaze
+//
+//  Created by Adam Issah on 8/3/22.
+//
+
+#import "RerunViewController.h"
+#import "Run.h"
+#import "RerunCell.h"
+#import "SceneDelegate.h"
+#import "HomeViewController.h"
+
+@interface RerunViewController () <UITableViewDataSource, UITableViewDelegate>
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+
+@end
+
+@implementation RerunViewController {
+    NSArray *pastRuns;
+}
+- (void) viewWillAppear:(BOOL)animated {
+    [Run retreiveRunObjects:PFUser.currentUser limit:10 completion:^(NSArray * _Nonnull runObjects, NSError * _Nullable err) {
```

Comment:
need to call `super` here

Reply:
I removed that function because we were performing the exact same action in viewDidLoad